fl4health/model_bases/fedsimclr_base.py

```diff
@@ -1,3 +1,6 @@
+from __future__ import annotations
```

Comment:
Is this used anywhere? I don't see where it is but I might be missing it.

Reply:
This is used to be able to return an object of type class from a method in class https://docs.python.org/3/library/__future__.html

Reply:
Oh cool. I didn't know that this existed! I've always just used the mypy escape mechanism of annotating in quotes as `-> "FedSimClrModel"`. I'll put a ticket in the backlog to add this in a few places where this is a little cleaner!

Reply:
Yeah! Eventually this will be part of the default functionality as far as I understand. They were supposed to have that in python 3.10 but it got pushed so you still have to use the from __future__